CHANGELOG.md

```diff
@@ -4,7 +4,7 @@ All notable changes to the sdc11073 module will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
```

Comment:
Would be nice to keep this, so that the next time it does not have to be added again, I would just leave it in everytime.